front/pages/w/[wId]/congratulations.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Button,
+  ConfettiBackground,
+  Page,
+  SparklesIcon,
+} from "@dust-tt/sparkle";
+import type { UserType, WorkspaceType } from "@dust-tt/types";
+import type { InferGetServerSidePropsType } from "next";
+import { useRouter } from "next/router";
+import { useRef } from "react";
+
+import OnboardingLayout from "@app/components/sparkle/OnboardingLayout";
+import config from "@app/lib/api/config";
+import { getUserMetadata } from "@app/lib/api/user";
+import { withDefaultUserAuthRequirements } from "@app/lib/iam/session";
+
+export const getServerSideProps = withDefaultUserAuthRequirements<{
+  user: UserType;
+  owner: WorkspaceType;
+  isAdmin: boolean;
+  defaultExpertise: string;
+  defaultAdminInterest: string;
+  conversationId: string | null;
+  gaTrackingId: string;
+  baseUrl: string;
+}>(async (context, auth) => {
+  const owner = auth.workspace();
+  const user = auth.user();
+
+  if (!owner || !user || !auth.isUser()) {
+    return {
+      redirect: {
+        destination: "/",
+        permanent: false,
+      },
+    };
+  }
+  const isAdmin = auth.isAdmin();
+  const expertise = await getUserMetadata(user, "expertise");
+  const adminInterest = isAdmin
+    ? await getUserMetadata(user, "interest")
+    : null;
+
+  // If user was in onboarding flow "domain_conversation_link"
+  // We will redirect to the conversation page after onboarding.
+  const conversationId =
+    typeof context.query.cId === "string" ? context.query.cId : null;
+
+  return {
+    props: {
+      user,
+      owner,
+      isAdmin,
+      defaultExpertise: expertise?.value || "",
+      defaultAdminInterest: adminInterest?.value || "",
+      conversationId,
+      baseUrl: config.getAppUrl(),
+      gaTrackingId: config.getGaTrackingId(),
+    },
+  };
+});
+
+export default function Congratulation({
+  owner,
+  conversationId,
+  gaTrackingId,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const router = useRouter();
+  const referentRef = useRef<HTMLDivElement>(null);
+  const redirectToApp = async () => {
+    if (conversationId) {
+      await router.push(`/w/${owner.sId}/assistant/${conversationId}`);
```

Comment:
`nit` I'd use replace to ensure customer can't go back to this page.
```suggestion
      await router.replace(`/w/${owner.sId}/assistant/${conversationId}`);
```